modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,296 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The {lcao} will do the following:
+
+* Checks for required system configurations
+* Performs any necessary system cleanup before generating the seed image
+* Launches the image generation tool, which does the following:
+** Stops the cluster operators
+** Prepares the seed image configuration
+** Generates and pushes the seed image to the image repository
```

Comment:
...to the image repository specified in the `imageBasedUpgrade` custom resource or in the repository specified by the user. IMHO I think that way we can link the information explained at the beginning of the section where you explain that there are two main CRs.